actions/insights/advisor_actions.py

```diff
@@ -142,6 +140,14 @@ def error(self, dispatcher: CollectingDispatcher, events):
         dispatcher.utter_message(response="utter_advisor_recommendation_pathways_error")
         return events
 
+    def resolve_recommendation_category(self, user_input):
+        for word in user_input.split(" "):
```

Comment:
I feel like ppl, might type "newrecs" and try to break it. But, this is good for now. :smile: